### PR TITLE
Button cleanup

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

An assortment of small Button component cleanup chores that will make it easier to implement the upcoming desktop theme. Details in the commits!

## Demo Video or Screenshot
N/A 

## Testing Plan
Automated tests plus some manual testing of VxMark

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/mark-flow-ui/src/components/review.tsx [hunk 1]
```diff
@@ -210,7 +204,7 @@ export function Review({
             footerAlign={selectionsAreEditable ? 'right' : undefined}
             footer={
               selectionsAreEditable && (
-                <DecoyButton>
+                <Button onPress={() => {}}>
```
Comment (adghayes) [inline libs/mark-flow-ui/src/components/review.tsx:207]: Based on your commit description, should you be passing a `tabIndex` here?

File: libs/mark-flow-ui/src/pages/start_page.tsx [hunk 2]
```diff
@@ -52,12 +50,6 @@ const Footer = styled.div`
   padding: 0.5rem;
 `;
 
-const LargeButtonText = styled.span`
-  font-size: 50px;
-  line-height: 2;
-  padding: 0 1rem;
-`;
-
```
Comment (adghayes) [inline libs/mark-flow-ui/src/pages/start_page.tsx:60]: I am confused by the styling change here:

1. Was this button really a set `px` before, and thus unresponsive to size settings?
2. From my reading, it looks like you made it responsive but a lot smaller, from `50px` to `1.2rem`? Is that right?

File: libs/ui/src/button.tsx [hunk 3]
```diff
@@ -28,15 +28,6 @@ export interface StyledButtonProps {
...
   rightIcon?: IconName | JSX.Element;
-
-  /** @deprecated Place the button within a flex or grid container instead. */
-  readonly fullWidth?: boolean;
-
-  /** @deprecated Incompatible with new VVSG size themes */
-  readonly large?: boolean;
-
-  /** @deprecated Incompatible with new VVSG size themes */
-  readonly small?: boolean;
```
Comment (adghayes) [inline libs/ui/src/button.tsx:39]: These were truly not doing anything at all? 

😮 

Scandalous.

File: libs/mark-flow-ui/src/components/review.tsx (hunk 1 above)
Comment (jonahkagan) [reply libs/mark-flow-ui/src/components/review.tsx:207]: Great catch, I think it got removed accidentally in a rebase

File: libs/mark-flow-ui/src/pages/start_page.tsx (hunk 2 above)
Comment (jonahkagan) [reply libs/mark-flow-ui/src/pages/start_page.tsx:60]: 1. Yes, as far as I can tell it was a set px before.
2. The font size with the default medium size theme now works out to 48px, so it's pretty much in the same range.

File: libs/ui/src/button.tsx (hunk 3 above)
Comment (jonahkagan) [reply libs/ui/src/button.tsx:39]: I think they were doing things long ago, but were deprecated and then unimplemented at some point along the way

**Review by adghayes — APPROVED** [review]

**Pull request merged**